### ISSUE TITLE
Support for .NET 6 RC1

### ISSUE THREAD

**Issue by tommykri** [Megabit/Blazorise#2944]
I'm trying to upgrade our project to .NET 6 RC1 using Blazorise v0.9.4.4. This works fine except for one issue.

When the circuit is disconnected because you are navigating away from the application, you will get this error for some components:

```
[Microsoft.AspNetCore.Components.Server.Circuits.RemoteRenderer] Unhandled exception rendering component: "JavaScript interop calls cannot be issued at this time. This is because the circuit has disconnected and is being disposed."
Microsoft.JSInterop.JSDisconnectedException: JavaScript interop calls cannot be issued at this time. This is because the circuit has disconnected and is being disposed.
   at Microsoft.AspNetCore.Components.Server.Circuits.RemoteJSRuntime.BeginInvokeJS(Int64 asyncHandle, String identifier, String argsJson, JSCallResultType resultType, Int64 targetInstanceId)
   at Microsoft.JSInterop.JSRuntime.InvokeAsync[TValue](Int64 targetInstanceId, String identifier, CancellationToken cancellationToken, Object[] args)
   at Microsoft.JSInterop.JSRuntime.InvokeAsync[TValue](Int64 targetInstanceId, String identifier, Object[] args)
   at Microsoft.JSInterop.JSRuntimeExtensions.InvokeVoidAsync(IJSRuntime jsRuntime, String identifier, Object[] args)
   at Blazorise.Button.DisposeAsync(Boolean disposing)
   at Blazorise.BaseAfterRenderComponent.DisposeAsync()
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.<>c__DisplayClass69_0.<<Dispose>g__HandleAsyncExceptions|1>d.MoveNext()
```

At least this happens for the following components:
- Button
- Modal
- Tooltip
- NumericEdit

Navigating away from the application would mean closing the tab, hard refresh (F5) or clicking an external link.

**Comment by stsrki** [comment]
We are still targeting an older .Net 6 preview and we still must update our nuggets packages. In fact I tried to do it last time, installed VS22, and had so many problems that nothing worked. So I reverted everything. Hopefully, the latest RC1 and VS will be much better work but we will see once I start. 🤞

**Comment by stsrki** [comment]
I have installed VS22 and RC1, and this time it was way fewer problems. So Blazorise is now updated to use the latest .Net 6. It will be available in a day or two with the next NuGet update.

**Comment by stsrki** [comment]
We still didn't publish new packages. The changes will be in 0.9.4.5 once we release them.

**Comment by tommykri** [comment]
Hi. I still do not think this issue have been resolved with 0.9.4.6 :)

Seeing a lot of exceptions when navigating away from the page:
![image](https://user-images.githubusercontent.com/5443815/136010828-20273623-c43a-4c81-98a3-225f6461d369.png)

For example a Button-component will throw an exception on this line because "JSDisconnectedException" isn't handled when disposing the component as per https://github.com/dotnet/aspnetcore/issues/33336#issuecomment-862425579:
https://github.com/Megabit/Blazorise/blob/67da3e1fb3256f83f25eb378026eda10768322b5/Source/Blazorise/Components/Button/Button.cs#L107

**Comment by stsrki** [comment]
You're right. I was naively thinking it would work with just upgrading .Net version. But the biggest problem is that they have also introduced a new Exception type for the JSInterop. https://github.com/dotnet/aspnetcore/issues/33336

For anyone looking into the problem, the solution is to also check for `JSDisconnectedException`

```
#if NET6_0_OR_GREATER
                catch ( Microsoft.JSInterop.JSDisconnectedException )
                {
                }
#endif
```

**Comment by stsrki** [comment]
It is now fixed and it will be available in 0.9.4.7

**Comment by tommykri** [comment]
Great, quick to resolve it as always :) Thanks!
